### ISSUE TITLE
Unable to place Item Translocator on top of Extra Utilities 2 Crusher

### ISSUE THREAD

**Issue by RyanP1991** [TheCBProject/Translocators#43]
Using version Translocators 1.12.2 - 2.5.1.77 I am unable to place item translocators on top of an extra utilities 2 crusher to input ore, although this worked correctly in the previous version 2.5.0.73. But I can now pull out of the machine okay so looks like something got changed in the update.

The items act as they get placed but in fact are ghost items and when you sort your inventory they reappear in it.

**Comment by nullKomplex** [comment]
Server admin here. This update is all sorts of screwy and I believe this is related. If not I'll make a new issue for you.

When we updated the server to 2.5.1.77 as soon as the server began ticking it immediately crashed:

https://gist.github.com/nullKomplex/76d857d3f497602184182d4567a018f6

I resolved this by removing the mod entirely, starting up the server so that it cleared all translocators, and re-adding the mod.

However, our console now gets spammed with this (snippet):

https://gist.github.com/nullKomplex/354aa9a5b6539b1512754ba61699df92

I highly suspect it's related.

Entire server log from Ryan's log in to disconnect (not sure the exact time he attempted to place the translocator(s)):

https://gist.github.com/nullKomplex/40e803001c55e69f157cc850a1817151
